scripts/qesap/lib/cmds.py

```diff
@@ -0,0 +1,307 @@
+"""
+sub commands library
+"""
+import os
+import shutil
+import re
+import yaml
+import logging
+log = logging.getLogger('QESAP')
+
+from lib.config import CONF
+import lib.process_manager
+from lib.status import Status
+
+
+def create_tfvars(config, template):
+    """ Create the tfvars file content
+
+    Args:
+        config (obj): CONF instance
+        template (str): tfvars template
+
+    Returns:
+        tfvar_content (dict): dictionary with tfvars content. None in case of error
+        err (str): Error message, None in case of PASS
+    """
+    if template:
+        log.debug("tfvar template %s", template)
+        tfvar_content = config.template_to_tfvars(template)
+        return tfvar_content, None
+    if config.terraform_yml():
+        log.debug("tfvar template not present")
+        tfvar_content = config.yaml_to_tfvars()
+        if tfvar_content is None:
+            log.error("Empty tfvar_content")
+            return None, "Problem converting config.yaml content to terraform.tfvars"
+        return tfvar_content, None
+    return None, "No terraform.tfvars.template neither terraform in the configuration"
+
+
+def create_hana_media(config_ansible, apiver):
+    """ Create the hana_media file content
+
+    Args:
+        apiver (int): value from apiver
+        config_ansible (dict): dictionary that rappresent the conf.yaml ansible section
+
+    Returns:
+        hanamedia_content (dict): dictionary with hana_media content. None in case of error
+        err (str): Error message, None in case of PASS
+    """
+    hanamedia_content = {}
+    if apiver < 3:
+        hana_url_re = r'http.*://(?P<ACCOUNT>.+)\.blob\.core\.windows\.net/(?P<CONTAINER>.+)/(?P<EXE>.+)'
```

Comment:
@alvarocarvajald the regexp that you commented ends up here

Reply:
Nice!